src/anupSJun20/ArmstrongNumber.java

```diff
@@ -0,0 +1,26 @@
+package anupSJun20;
+/*Program 4: WAP to check if a number is Armstrong or not.
+Example:-
+Input:- 153
+Output:-153
+hint:- (1*1*1)+(5*5*5)+(3*3*3)
+1+125+27
+153*/
+
+public class ArmstrongNumber {
+	static void findANumberArmstrong(int num) {
+		int input = num;
+		double newNum = 0;
+		while(num>0) {
+			newNum = newNum + Math.pow((num%10), 3);
```

Comment:
There is no need to change data type of newNum from int to double due to power function.
you can use typecasting here.
newNum = (int) (newNum + (Math.pow((number % 10), 3)));